lib/go/test/flow_idtable_staking_test.go

```diff
@@ -776,16 +781,65 @@ func TestIDTableApprovals(t *testing.T) {
 		1,
 		false)
 
-	nodeIDs := make([]string, 1)
-	nodeIDs[0] = adminID
+	nodeIDs := make([]string, 2)
+	nodeIDs[0] = accessID
+	nodeIDs[1] = adminID
 	nodeIDDict := generateCadenceNodeDictionary(nodeIDs)
-	initialNodeIDs := cadence.NewArray([]cadence.Value{CadenceString(adminID)}).WithType(cadence.NewVariableSizedArrayType(cadence.NewStringType()))
+	initialNodeIDs := cadence.NewArray([]cadence.Value{CadenceString(adminID), CadenceString(accessID)}).WithType(cadence.NewVariableSizedArrayType(cadence.NewStringType()))
 
-	t.Run("Should be able to add a node to approved node list", func(t *testing.T) {
+	// Update the access node minimum to zero
+	tx := createTxWithTemplateAndAuthorizer(b, templates.GenerateChangeMinimumsScript(env), idTableAddress)
+	err := tx.AddArgument(cadence.NewArray([]cadence.Value{CadenceUFix64("250000.0"), CadenceUFix64("250000.0"), CadenceUFix64("1250000.0"), CadenceUFix64("135000.0"), CadenceUFix64("0.0")}))
+	require.NoError(t, err)
 
-		tx := createTxWithTemplateAndAuthorizer(b, templates.GenerateSetApprovedNodesScript(env), idTableAddress)
+	signAndSubmit(
+		t, b, tx,
+		[]flow.Address{idTableAddress},
+		[]crypto.Signer{IDTableSigner},
+		false,
+	)
 
-		err := tx.AddArgument(nodeIDDict)
+	// Register an access node
+	amountToCommit = 00000000
```

Comment:
is it this many zeros on purpose?

Reply:
The UFix64 values have 8 decimal places, so that is how I was initializing them to be zero. I guess it could probably just be one zero for this value since it is mean to just be zero